package.json

```diff
@@ -134,12 +137,17 @@
   "scripts": {
     "prepare": "npm run compile-prod",
     "compile": "node ./build.js",
-    "compile-prod": "NODE_ENV=production ./build.sh && ls -lh ./lib/web/js && echo \"DONE\"",
+    "compile-prod": "cross-os compile-prod",
     "test": "cross-os test",
     "watch": "npm-watch",
     "precommit": "lint-staged"
   },
   "cross-os": {
+    "compile-prod": {
+      "darwin": "NODE_ENV=production ./build.js && ls -lh ./lib/web/js && echo \"DONE\"",
+      "linux": "NODE_ENV=production ./build.js && ls -lh ./lib/web/js && echo \"DONE\"",
+      "win32": "SET NODE_ENV=production&& node ./build.js && ls -lh ./lib/web/js && echo \"DONE\" && SET NODE_ENV="
```

Comment:
is the `production&&` <-- `&&` purposely not spaced?

Reply:
@slvnperron correct, in Windows, if you leave a space, then NODE_ENV = "production "
so the lack of space is required